lib/colony.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Colony do
```

Comment:
as a matter of convention generally we submodule under your application, so this would probably be `AntColony.Colony` and in the subdirectory `lib/ant_colony